Makefile

```diff
@@ -14,6 +14,13 @@ requirements: scripts/_vendor/pip-post-compile.sh
 	pip-compile -o requirements/docs-requirements.txt requirements/docs-requirements.in
 	bash scripts/pip-post-compile.sh requirements/*requirements.txt
 
+	cp requirements/*requirements.txt requirements-python3_6/
+	grep -v '^futures==' requirements-python3_6/requirements.txt > temp && mv temp requirements-python3_6/requirements.txt
```

Comment:
These would probably be more readable with `sed '/^futures==/d' filename.txt` https://ccm.net/faq/1451-delete-lines-from-a-file-using-sed

Reply:
Yeah, I was following the example in ```scripts/pip-post-compile.sh``` but I think it's worth updating in both places (will do follow-up PR).